lib/elixir_boilerplate_web/router.ex

```diff
@@ -15,6 +15,13 @@ defmodule ElixirBoilerplateWeb.Router do
     plug(:put_root_layout, {ElixirBoilerplateWeb.Layouts, :root})
   end
 
+  scope "/" do
+    pipe_through(:browser)
+    # Metrics can contains sensitive data you should protect it under authorization
+    # See https://github.com/mirego/telemetry_ui#security
+    get("/metrics", TelemetryUI.Web, [], assigns: %{telemetry_ui_allowed: true})
```

Comment:
Should we `IO.warn/1` this message to the console so developer see an explicit message about securing the endpoint?

```suggestion
    # Metrics can contains sensitive data you should protect it under authorization
    # See https://github.com/mirego/telemetry_ui#security
    IO.warn("⚠️ TelemetryUI endpoint might need to be secured for production environnement!")
    get("/metrics", TelemetryUI.Web, [], assigns: %{telemetry_ui_allowed: true})
```

Reply:
That will break the linting task I think because our code warns on compile

Reply:
Well we can either add a `credo` directive to disable the check, or add a section in the boilerplate `README` about security concerns to keep in mind before going to production.

Reply:
We could use `telemetry_ui_allowed: false` and add a comment to explicitely enable it wisely.

Reply:
Files are updated with your comments